src/react-apps/applications/service-development/src/types/global.ts

```diff
@@ -1,14 +1,83 @@
+import { IHandleServiceInformationState } from '../features/administration/handleServiceInformationReducer';
 import { IHandleMergeConflictState } from '../features/handleMergeConflict/handleMergeConflictReducer';
 import { IFetchedLanguageState } from '../utils/fetchLanguage/languageReducer';
 
 declare global {
-  export interface IServiceDevelopmentNameSpace<T1, T2> {
+  export interface IServiceDevelopmentNameSpace<T1, T2, T3> {
     language: T1;
     handleMergeConflict: T2;
+    serviceInformation: T3;
   }
 
   export interface IServiceDevelopmentState
     extends IServiceDevelopmentNameSpace
     <IFetchedLanguageState,
-    IHandleMergeConflictState> { }
+    IHandleMergeConflictState,
+    IHandleServiceInformationState> { }
+}
+
+export interface IRepository {
+  clone_url: string;
+  created_at: string;
+  default_branch: string;
+  description: string;
+  empty: boolean;
+  fork: boolean;
+  forks_count: number;
+  full_name: string;
+  html_url: string;
+  id: number;
+  mirror: boolean;
+  name: string;
+  open_issues_count: number;
+  owner: IOwner;
+  permissions: IPermissions;
+  private: boolean;
+  size: number;
+  ssh_url: string;
+  stars_count: number;
+  updated_at: string;
+  watchers_count: number;
+  website: string;
+  is_cloned_to_local: boolean;
```

Comment:
Minor comment: Sort the last two entries in alphabetical order.

Reply:
sorted